interface/src/app/node/node.js

```diff
@@ -73,12 +78,15 @@ angular.module('adage.node', [
               sysName = response.objects[i].gene.systematic_name;
               stdName = response.objects[i].gene.standard_name;
               desc = response.objects[i].gene.description;
+              entrezID = response.objects[i].gene.entrezid;
               if (desc.toLowerCase() === 'hypothetical protein') {
                 ++numHypo;
               }
               $scope.genes.push(
-                {sysName: sysName, stdName: stdName, desc: desc});
+                {sysName: sysName, stdName: stdName, desc: desc,
+                 entrezID: entrezID});
             }
+            $scope.hWGenes = $scope.genes;
```

Comment:
Since "hWGenes" and "genes" are equal in the scope, do you think it makes sense to merge these variables into one?